### DISCUSSION TITLE
Skywalking compatibility with Skywalking 9.5.0 and Satellite version 1.12

### DISCUSSION THREAD

**Discussion by skundu1234** [apache/skywalking#10976] [General]
We have seen that skywalking 9.5.0 is compatible with Satellite 1.12 but dont find in github for satellite version of 1.12 ,its there 1.11.

Please let me know is this document incorrect or planning to enable skywalking satellite 1.12
<img width="1758" alt="Screenshot 2023-06-21 at 12 15 58 PM 1" src="https://github.com/apache/skywalking/assets/87635275/fce1e154-37d1-4f11-aa21-4543316c1929">

<img width="1574" alt="Screenshot 2023-06-21 at 12 14 18 PM" src="https://github.com/apache/skywalking/assets/87635275/a37d21e1-fa95-40fb-8871-e82640e5d632">

**Comment by wu-sheng** [comment]
Check mail list, it is on voting.

**Reply by wu-sheng** [reply]
And it is 1.2, not 1.12

**Reply by skundu1234** [reply]
Okay ,so can it support satellite >=1.0.0 like pervious skywalking version as Satellite 1.2.0 is still INPRG and still not live

**Reply by wu-sheng** [reply]
The doc is very clear. Others may or may not work are not from official.

**Reply by skundu1234** [reply]
Thanks @wu-sheng

**Reply by skundu1234** [reply]
I can see tag 1.2.0 and when it will be officially released any tentative date, please let me know.So we can plan for our production upgrade with 9.5.0.
Thanks in advance @wu-sheng

**Reply by wu-sheng** [reply]
That is a community vote. It is when the release manager continues tge process.
I don't have control on that.
